ABB.SrcML.Test/LibSrcMLRunnerTests.cs

```diff
@@ -0,0 +1,481 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.IO;
+using NUnit.Framework;
+using ABB.SrcML;
+using System.Xml;
+using System.Runtime.InteropServices;
+using System.Xml.Linq;
+
+namespace ABB.SrcML.Test {
+
+    [TestFixture]
+    [Category("Build")]
+    class LibSrcMLRunnerTests {
+        private const string TestInputPath = @"..\..\TestInputs";
+        [TearDown]
+        public void Cleanup() {
+            File.Delete("output0.cpp.xml");
+            File.Delete("output1.cpp.xml");
+        }
+        /// <summary>
+        /// This tests the creation of an archive using a list of source files
+        /// </summary>
+        [Test]
+        public void TestCreateSrcMLArchiveFtF() {
+            LibSrcMLRunner.SourceData ad = new LibSrcMLRunner.SourceData();
+            ad.SetArchiveFilename(Path.Combine(TestInputPath, "input.cpp"));
+            ad.SetArchiveLanguage(LibSrcMLRunner.SrcMLOptions.SRCML_LANGUAGE_CXX);
+
+            LibSrcMLRunner.SourceData bc = new LibSrcMLRunner.SourceData();
+            bc.SetArchiveFilename(Path.Combine(TestInputPath, "input2.cpp"));
+            bc.SetArchiveLanguage(LibSrcMLRunner.SrcMLOptions.SRCML_LANGUAGE_CXX);
+
+            List<LibSrcMLRunner.SourceData> data = new List<LibSrcMLRunner.SourceData>();
```

Comment:
It looks like the List<> "data" is never used and can be removed. There is at least one other test with this same pattern.